package.json

```diff
@@ -69,7 +69,7 @@
     "@testing-library/react": "^10.0.3",
     "@testing-library/react-hooks": "^3.2.1",
     "@testing-library/user-event": "^10.3.1",
-    "@types/react": "^16.8.0",
+    "@types/react": "16.9.23",
```

Comment:
we *may* even be able to drop this. what does everyone think?

Reply:
Don't we need this for types from React like `HTMLAttributes`, at least that was my understanding from https://www.typescriptlang.org/docs/handbook/react-&-webpack.html#install-our-dependencies? (those instructions are for webpack but I figure the package.json stuff applies universally)

Reply:
yes, seems so 👍